keyboards/creatkeebs/thera/config.h

```diff
@@ -29,6 +29,9 @@
 /* COL2ROW or ROW2COL */
 #define DIODE_DIRECTION COL2ROW
 
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
```

Comment:
The default is 5 anyway, so in theory this change wont really do anything?

https://github.com/qmk/qmk_firmware/blob/ab1898e6600fe0f4f4a74ac26cb2d25c6f736161/quantum/debounce/sym_defer_g.c#L24-L26